pkg/worker/stage/stage.go

```diff
@@ -110,12 +138,37 @@ func (sm *stageManager) ExecCodeCheckout(token string, stage *api.CodeCheckoutSt
 	if err != nil {
 		return err
 	}
+	formatPipelineRecordName(commitID)
+
+	repoName, errn := scm.GetRepoName(codeSource)
+	if errn != nil {
+		log.Warningf("get repo name fail %s", errn.Error())
+	}
+	commitLog := scmProvider.GetTagCommitLog(cloneDir, "master")
```

Comment:
We should support other branches and tags.

Reply:
done